<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpCfger</class>
 <widget class="QFrame" name="UdpCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>130</height>
   </rect>
  </property>
  <widget class="QLabel" name="localAddrTag">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Source address:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="remotePortTag">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QIpAddrEditor" name="remoteAddr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
  </widget>
  <widget class="QLabel" name="remoteAddrTag">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination address:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="remotePort">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1024</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
   <property name="value">
    <number>5050</number>
   </property>
  </widget>
  <widget class="QComboBox" name="localAddr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIpAddrEditor</class>
   <extends>QFrame</extends>
   <header>QIpAddrEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
